skel/Makefile

```diff
@@ -0,0 +1,82 @@
+SHELL := /bin/bash
+
+.PHONY: elm assets watch minify
+
+default: all
+
+NPM_PATH := ./node_modules/.bin
+ASSETS_DIR := ./lib/assets
+ELM_DIR := ./lib/elm
+SCSS_DIR := ./lib/scss
+
+export PATH := $(NPM_PATH):$(PATH)
+
+all: elm scss assets
+
+assets:
+	@mkdir -p lib/dist/assets/ && cp -R ${ASSETS_DIR} lib/dist/
+
+build:
+	make deps
+	$(MAKE) test -C ${ELM_DIR}
+	make all
+	make minify
+
+clean:
+	@rm -Rf dist/*
+
+deps:
+	@npm install
+	$(MAKE) deps -C ${ELM_DIR}
+	$(MAKE) deps -C ${SCSS_DIR}
+
+distclean: clean
+	@rm -Rf node_modules
+	$(MAKE) distclean -C ${ELM_DIR}
+	$(MAKE) distclean -C ${SCSS_DIR}
+
+elm:
+	$(MAKE) -C ${ELM_DIR}
+	@cp ${ELM_DIR}/dist/main.js lib/dist/
+
+help:
+	@echo "Run: make <target> where <target> is one of the following:"
+	@echo "  all                    Compile all Elm and Scss files"
+	@echo "  build                  Install deps and compile for production"
+	@echo "  clean                  Remove 'dist' folder"
+	@echo "  deps                   Install build dependencies"
+	@echo "  distclean              Remove build files and dependencies"
+	@echo "  help                   Show list of make commands"
+	@echo "  init                   Initialize site, clone Elm and Scss skeletons and install deps"
+	@echo "  minify                 Minify JS and Css files"
+	@echo "  watch                  Compile all Elm and Scss files on file change and reload browser"
+	@echo "  watch-elm              Compile all Elm files on file change and reload browser"
+	@echo "  watch-scss             Compile all Scss files on file change and reload browser"
```

Comment:
Some targets, such as `elm`, `assets`, `livereload` and `scss` are missing from this list.

Reply:
Fixed